.php_cs.dist

```diff
@@ -26,7 +26,7 @@ $finder = PhpCsFixer\Finder::create()
 return PhpCsFixer\Config::create()
     ->setRules([
         '@PSR2' => true,
-        'array_syntax' => [ 'syntax' => 'short' ],
+        'array_syntax' => [ 'syntax' => 'long' ],
```

Comment:
Could you revert this one back to short? It provides lots of unnecessary changes in the code, unrelated to the PR

Reply:
`As of PHP 5.4 you can also use the short array syntax, which replaces array() with [].`
If php5.3 is not considered, it can be restored.

Reply:
Ah, good catch! We should keep it then, and perhaps only move on to newer (supported) PHP versions only with a new major release (2.x)